README.md

```diff
@@ -128,12 +91,50 @@ In order to understand how Flow works, please read [documentation](src/core/etl/
 * Pipeline - Interface representing ETL process, each received `Rows` instanced is passed through all `Pipes`, also responsible for error handling.
 * Pipe - Loader of Transformer instance existing in `Pipes` collection.
 
-### Supported PHP versions
+## Contributing
```

Comment:
Maybe extract this into a `CONTRIBUTING.md` file, like:
- https://github.com/symfony/symfony/blob/6.4/CONTRIBUTING.md
- https://github.com/meilisearch/meilisearch-php/blob/main/CONTRIBUTING.md
- https://github.com/api-platform/core/blob/main/CONTRIBUTING.md

Then we can extend it, but it will reduce the size of the `README` file.